back/app/services/participants_service.rb

```diff
@@ -101,6 +101,28 @@ def project_participants(project, options = {})
     User.where(id: participant_ids)
   end
 
+  # Returns the total count of project participants including anonymous posts
+  def project_participants_count(project)
+    Rails.cache.fetch("#{project.cache_key}/participant_count", expires_in: 1.hour) do
+      participant_ids = Rails.cache.fetch("#{project.cache_key}/participant_ids", expires_in: 1.day) do
```

Comment:
Do you know why the version in `project_participants` passes through the options?

Reply:
No, the options array doesn't seem to be used anywhere in the codebase. I should probably remove entirely - but at least remove it from caching as it it's called differently in different places the cache behaviour could screw up

Reply:
Okay, yeah fine for me to leave as it is. But if the options are not doing anything, perhaps we could consider removing them at some point. I also don't feel like the caching key should be the same if the options could be different.

Reply:
I removed them from this method as they were clearly never used